index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./index.css">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    <script src="https://kit.fontawesome.com/4905cce8d8.js" crossorigin="anonymous"></script>
+    <title>my-portfolio-microverse</title>
+</head>
+
+<body>
+    <header class="header">
+        <nav class="nav">
+            <h3><a href="">Kigozi Joseph</a></h3>
+            <i class="nav-item fas fa-bars fa-lg"></i>
+        </nav>
+    </header>
+    <div>
+
+        <section class="catch-text-sec">
+            <h2 id="catch-text-heading">I'm Kigozi<br/> glad to see <br/> you!</h2>
```

Comment:
[OPTIONAL] 👍🏽 

- I think it will be better for your  `<h2 id="catch-text-heading">` to be an `<h1>`  tag so that Search engines, just like your readers can look at your H1s and understand about your page and its content. Also, the h1 defines your page's main title, header and helps Search engine optimization.

Reply:
thanks @Favourezeugwa

Reply:
Ill change it ryt away